docusaurus/docs/introduction/plugin-lifecycle.md

```diff
@@ -0,0 +1,93 @@
+
+:::note
+
+The life cycle of plugins is tracked in-memory and is not persisted in Grafana’s database. 
+
+:::
+
+## Installing and uninstalling a plugin
+
+For instructions to install or uninstall plugins, see our documentation at [Plugin administration](https://grafana.com/docs/grafana/latest/administration/plugin-management/#install-grafana-plugins).
+
+Upon installation, a plugin is extracted to the _plugin directory_ on the filesystem. Similarly, uninstalling a plugin deletes the files from the same directory.  
+
+:::note
+
+Use of the Grafana CLI to install or uninstall a plugin requires you to restart Grafana for the change to take effect. To avoid restarting Grafana, you can install a plugin from within the Grafana [plugin catalog](https://grafana.com/plugins/) during runtime.
+
+:::
+
+## Loading plugins
+
+Understanding the different phases involved when Grafana is starting a plugin may help you better understand plugin usage and to debug any unexpected behavior during loading.
+
+### Phase 1. Plugin initialization
+
+All plugins are initialized either when Grafana starts up or when a plugin has been installed/uninstalled during runtime.
+
+For [backend](./backend.md) plugins, there is an additional initialization process (see [Phase 4](#phase-4-backend-plugin-initialization)).
+
+### Phase 2. Plugin discovery 
+
+Grafana starts to discover which plugins are installed by scanning directories on the file system for every `plugin.json`.
+
+### Phase 3. Plugin loading 
+
+All plugins that were discovered in the discovery phase are checked to make sure they’re valid and that they have a valid signature. Valid plugins are referred to as _verified plugins_. 
+
+### Phase 4. Backend plugin initialization
+
+For any verified plugin that has a backend, Grafana configures a backend client. Depending on the plugin's classification, one of these things will happen: 
+- **Core plugins** - the backend client is already configured and nothing more needs to be done.
+- **External plugins** - Grafana configures the backend client to use HashiCorp’s Go Plugin System over RPC.
+
+### Phase 5. Registration
+
+All verified plugins are registered in an in-memory registry. From now on, the plugin is available within Grafana and so are referred to as _registered plugins_.
+
+### Phase 6. Start backend plugin (backend plugins only)
+
+For registered external plugins that have a backend, Grafana starts to run the backend binary as a separate process using HashiCorp’s Go Plugin System over RPC. The supported plugin protocol and version is negotiated between Grafana (client) and the plugin (server) to give Grafana an understanding of the plugin's capabilities.  
+
+A Grafana backend plugin has its own separate life cycle. So long as the backend plugin is running, Grafana will make sure to restart the backend plugin in case it crashes or is killed. When Grafana is shut down, the backend processes is then terminated.  
```

Comment:
Less hard phrasing perhaps :) 

```suggestion
A Grafana backend plugin has its own separate life cycle. So long as the backend plugin is running, Grafana will make sure to restart the backend plugin in case it crashes or is terminated. When Grafana is shut down, the backend processes is then terminated.  
```

Reply:
good catch.